### ISSUE TITLE
Missing release

### ISSUE THREAD

**Issue by sondrelg** [pydantic/pydantic-settings#90]
Hi, 

Sorry if this is reported just before you're intending to do a release. If so please just close 👍 

I've just used Kludex's codemod to upgrade my project. As a consequence I noticed I need to install `pydantic-settings` as a new dependency. However, as the latest `pydantic-settings` release sets a requirement on Pydantic `v2.0a3` (fixed 4 days ago, but not release), and since the FastAPI v0.100 alpha release uses a feature introduced in v2.0a4, the two are currently incompatible.

Not sure how clear that was, but just wanted to give a heads up that loosening the requirement on pydantic version seems necessary to make this work with the FastAPI pre-release 👍 Thanks!

**Comment by hramezani** [comment]
Hi,
`pydantic-settings` is in pre-release(beta) and the latest version is [2.0b2](https://pypi.org/project/pydantic-settings/2.0b2/)
which depends on `pydantic==2.0b3` .

Have you tried to install `pydantic-settings` by `pip install pydantic-settings --pre`?

I've installed it in a fresh env and here is the output of `pip freeze`:

```
annotated-types==0.5.0
pydantic==2.0b3
pydantic-settings==2.0b2
pydantic_core==0.39.0
python-dotenv==1.0.0
typing_extensions==4.7.0rc1
```

**Comment by sondrelg** [comment]
Ah, I missed the pre-releases. Version 1.99 depends on pydantic v2.0a3, according to Poetry

**Comment by sondrelg** [comment]
This seems fine though, as long as there's pre-releases available with >2.0a3. Thanks for the swift reply 🙇
